Assets/Project/Modules/VFX/Generic/Scripts/ParticleBehaviours/InterpolatorRecycleParticle.cs

```diff
@@ -11,24 +11,33 @@ namespace Popeye.Modules.VFX.Generic.ParticleBehaviours
     public class InterpolatorRecycleParticle : RecyclableObject
     {
         [SerializeField] internal bool _interpolateOnInit;
+        [SerializeField] private float _despawnDelay = 0.0f;
         [SerializeField] internal InterpolatorRecycleParticleData[] _interpolations;
-        
+
         [Header("LIGHT")]
         [SerializeField] private Light _light;
         [SerializeField] private float _duration;
         [SerializeField] private float _intensityGoal;
+        [SerializeField] private Ease _lightEase = Ease.Linear;
```

Comment:
There are some variables that could be incapsulated inside a Configuration ScriptableObject. I would demand this change, but since we are short on time and you would need to adapt all particles, I'll let it through. + I should have demanded it way earlier in the production in any case, that is my bad.